mixer/example/servicegraph/Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: docker
```

Comment:
Would it be cleaner to have one central Makefile instead of scattered ones?
Is this sub-project still maintained to matter?

Reply:
it is maintained.